pylammpsmpi/helpers/callbacks.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH
```

Comment:
@raynol-dsouza This helper class is only used in the tests or is this something you want to include in the pylammpsmpi package?

Reply:
> @jan-janssen I wrote a unit test for `set_fix_external_callback`, but I absolutely do not like the way it currently works. I tried a bunch of things, but the only thing that allowed the test to access the callback function and a class instance was if it was in a separate directory inside of the main package. Would appreciate some wisdom on the best way forward!

Only for the tests.